### DISCUSSION TITLE
我有个问题，如果我的构建结果将build产物忽略是不是lastUpdated 就是错误的，都是生成那一刻的时间？

### DISCUSSION THREAD

**Discussion by lzwcyd** [vuepress/core#1476] [Q&A] [answered]
我有个问题，如果我的构建结果将build产物忽略是不是lastUpdated 就是错误的，都是生成那一刻的时间？

**Comment by Mister-Hope** [comment]
我也有一个问题，你能清晰准确表达你想要问什么么？

什么叫构建结果忽略构建产物？

**Reply by lzwcyd** [reply]
不好意思我的描述不太清晰，我的意思是，运行yarn doc:build得到的构建结果目录被我放到gitignore里面了，我的git并不会追踪这些构建产物，然后出现了一种情况，所有的页面时间都变成了我最后一次更新的时间，而不是每个页面最后更新的时间，我的仓库地址如下：https://github.com/lzwcyd/lzwcyd.github.io

**Reply by Mister-Hope** [reply, marked as answer]
To get the correct modify time, you need to fetch the whole history

https://github.com/actions/checkout?tab=readme-ov-file#Fetch-all-history-for-all-tags-and-branches

I believed the docs already warn you at https://v2.vuepress.vuejs.org/guide/deployment.html#github-pages

**Reply by lzwcyd** [reply]
谢谢，是我没有仔细查看示例，很抱歉打扰您
